### PR TITLE
[yolov3] Fixing protobuf API. (WIP)

### PR DESCRIPTION
**Pull request by arturgontijo**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] CircleCI tests are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md](https://dev.singularitynet.io/docs/contribute/contribution-guidelines/) document are followed

- Fix for #62 
- DO NOT MERGE! (First we need the fix for the `snet-dapp` component)

### CODE REVIEW

File: services/yolov3-object-detection/service/service_spec/object_detection.proto [hunk 1]
```diff
@@ -6,12 +6,19 @@ message Input {
     string confidence = 3;
 }
 
+message Boxes {
+    float x = 1;
+    float y = 2;
+    int32 w = 3;
```
Comment (vsbogd) [inline services/yolov3-object-detection/service/service_spec/object_detection.proto]: Minor, but seems a bit strange that `x` and `w` or `y` and `h` has different types.

Comment (arturgontijo) [reply services/yolov3-object-detection/service/service_spec/object_detection.proto]: `x` and `y` come from a division by 2 ([here](https://github.com/singnet/dnn-model-services/blob/master/services/yolov3-object-detection/service/object_detection.py#L97)) but I agree with you.
I'll `round()` them and change the `.proto` to `int32`.
Thanks!

**Comment by vsbogd** [comment]
Excuse me @arturgontijo , what is the reason of closing it?

**Comment by arturgontijo** [comment]
Sorry about that @vsbogd , was a dummy mistake!
I've deleted my fork branch and it reflected here...I've forgot that this PR was still open!
Will restore my branch and re-open it...and then will ask platform's team to update the `.proto` file on Mainnet (this was the rease why this PR was still open at first =/).

**Pull request closed without merging**